docs/architecture/overview.md

```diff
@@ -10,10 +10,11 @@ At the heart of Kubeapps is an in-cluster Kubernetes dashboard that provides you
 
 The dashboard is written in the JavaScript programming language and is developed using the React JavaScript library.
 
-### Kubeops
+### Kubeapps-APIs
 
-Kubeops is the service in charge of communicating both with the Helm (v3) API and other k8s resources like AppRepositories or Secrets.
```

Comment:
I would remove Kubeops from the docs once we have also removed the code. I mean, if someone sees Kubeapps is deploying a "kubeops" thing, and it even does not appear on the docs, it's quite odd.

I'd rather append the kubeapps-apis section to the list, and, perhaps, would add a _deprecate_ note on the kubeops item?

Reply:
OK, I'll update.